plugins/modules/vmware_dvswitch_info.py

```diff
@@ -0,0 +1,314 @@
+                                'numPorts': switch_obj.summary.numPorts,
+                                'ioControl': switch_obj.config.networkResourceManagementEnabled,
+                            },
+                            'advanced': {
+                                'maxMtu': switch_obj.config.maxMtu,
+                                'multicastFilteringMode': switch_obj.config.multicastFilteringMode,
+                            },
+                            'discoveryProtocol': {
+                                'protocol': switch_obj.config.linkDiscoveryProtocolConfig.protocol,
+                                'operation': switch_obj.config.linkDiscoveryProtocolConfig.operation,
+                            },
+                            'administratorContact': {
+                                'name': switch_obj.config.contact.name,
+                                'contact': switch_obj.config.contact.contact
+                            }
+                        },
+                        'privateVlan': pvlans,
+                        'netflow': {
+                            'switchIpAddress': switch_obj.config.switchIpAddress,
+                            'collectorIpAddress': switch_obj.config.ipfixConfig.collectorIpAddress,
+                            'collectorPort': switch_obj.config.ipfixConfig.collectorPort,
+                            'observationDomainId': switch_obj.config.ipfixConfig.observationDomainId,
+                            'activeFlowTimeout': switch_obj.config.ipfixConfig.activeFlowTimeout,
+                            'idleFlowTimeout': switch_obj.config.ipfixConfig.idleFlowTimeout,
+                            'samplingRate': switch_obj.config.ipfixConfig.samplingRate,
+                            'internalFlowsOnly': switch_obj.config.ipfixConfig.internalFlowsOnly
+                        },
+                        'healthCheck': health_check
+                    },
+                    'hosts': host_members,
+                    'folder': switch_obj.parent.name
+                }
+            })
+
+        self.module.exit_json(changed=False, distributed_virtual_switches=distributed_virtual_switches)
+
+    def properties_facts(self):
+        distributed_virtual_switches = []
+        for switch_obj in self.switch_objs:
+            distributed_virtual_switches.append(self.to_json(switch_obj, self.params.get('properties')))
+
+        self.module.exit_json(changed=False, distributed_virtual_switches=distributed_virtual_switches)
+
+
+def main():
+    argument_spec = vmware_argument_spec()
+    argument_spec.update(
+        folder=dict(type='str', required=False),
+        switch_name=dict(type='str', required=False, aliases=['switch', 'dvswitch']),
+        schema=dict(type='str', choices=['summary', 'vsphere'], default='summary'),
+        properties=dict(type='list', required=False, elements='str')
+    )
+
+    module = AnsibleModule(argument_spec=argument_spec, supports_check_mode=False)
```

Comment:
Info modules always support check_mode. So

```suggestion
    module = AnsibleModule(argument_spec=argument_spec, supports_check_mode=True)
```

Also, add tests for check_mode.